Countly.d.ts

```diff
@@ -126,6 +126,46 @@ declare module "countly-sdk-react-native-bridge" {
       export function reportFeedbackWidgetManually(widgetInfo: FeedbackWidget, widgetData: object, widgetResult: RatingWidgetResult | object): Promise<ErrorObject>;
     }
 
+    /**
+     * Countly Event Module
+     */
+    namespace event {
+      /**
+       * Used to send various types of event;
+       *
+       * @param {CountlyEventOptions} options event
+       * @return {string | void} error message or void
+       */
+      export function sendEvent(options: CountlyEventOptions): string | void;
```

Comment:
I thing we want to go with recordEvent instead of sendEvent if possible

Reply:
alright

Reply:
done